src/pages/eats/dashboard.hbs

```diff
@@ -0,0 +1,143 @@
+            {{> bellIcon class="d-none d-lg-block me-4 bxi-icon-color-secondary" }}
+            <p id="username-container" class="bxi-color-primary fw-bold d-none d-sm-flex mb-0 pt-0 me-2 mt-lg-0">{{settings.dashboard.username}}</p>
+            <div class="bxi-user-icon d-flex d-lg-none bxi-bg-primary" data-bs-toggle="collapse" data-bs-target=".bxi-auth-buttons">
+              {{> userIcon }}
+            </div>
+            <div class="d-none d-lg-flex static-user-icon me-2">
+              {{> userIcon }}
+            </div>
+            <div class="bxi-auth-buttons bxi-auth-buttons--secondary-link bxi-bg-primary bxi-zi-2 pb-1 pb-lg-0 bxi-bg-lg-transparent collapse navbar-collapse">
+              <button onclick="bxi.logout()" class="bxi-button mb-1 mb-lg-0">Log Out</button>
+            </div>
+          </div>
+        </div>
+      </nav>
+    </header>
+    <section class="bxi-dashboard position-relative d-flex align-items-end">
+      <div class="container">
+        <h1 class="bxi-ff-secondary bxi-main-header fw-bold mb-2 mt-5">{{settings.dashboard.title}}</h1>
+        <h2 class="bxi-color-primary bxi-ff-secondary fw-bold mb-4">{{settings.dashboard.description}}</h2>
+        <div class="row bxi-stretch-third-md">
+          {{#each settings.dashboard.categories}}
+          <div class="col-md-6 col-lg-4">
+            <div class="bxi-categories mb-4 d-flex p-4 flex-column align-items-start overflow-hidden position-relative">
+              <div class="img-wrapper"><img class="position-absolute top-50 start-50 translate-middle bxi-img-cover" src="{{background_image}}" /></div>
+              <h1 class="position-relative bxi-categories-section--text bxi-zi-2 fw-bold mb-0">{{percentage}}</h1>
+              <h5 class="position-relative bxi-categories-section--text bxi-zi-2 fw-bold mb-5">{{category}}</h5>
+              <button class="bxi-button bxi-bg-primary fw-bold position-relative mt-5 mb-4 py-3 px-5">{{button}}</button>
+            </div>
+          </div>
+          {{/each}}
+        </div>
+        <div class="row mt-5">
+          <div class="col-lg-7 col-xl-8 mb-5">
+            <div class="bxi-eats-portal rounded-2 mb-4 p-2">
+              <h1 class="bxi-eats-portal--header bxi-ff-secondary mt-3 ms-4 mb-4">{{settings.dashboard.portal.title}}</h1>
+              {{#each settings.dashboard.portal.orders}}
+              <div class="d-flex justify-content-between bxi-checklist-item py-3 px-5">
+                <div class="d-flex align-items-center">
+                  <div class="bxi-checkmark d-flex justify-content-center align-items-center rounded-circle bxi-checkmark--checked">
+                    {{> checkmarkIcon}}
+                  </div>
+                  <div class="ms-4">
+                    <h5 class="bxi-eats-dashboard--color fw-bold mb-0">{{date}}</h5>
+                    <p class="bxi-eats-portal--text mb-0">{{items}}</p>
+                  </div>
+                </div>
```

Comment:
Can you make this block here a real button since it could be actionable?

```
<button class="bxi-document bxi-button bxi-button--link d-flex flex-nowrap align-items-center">
  {{> documentIcon class='me-0'}}
  <span class="d-inline-block ms-2 bxi-document--color ms-0">{{button}}</span>
</button>
```